ecs-logs-collector.sh

```diff
@@ -235,28 +262,29 @@ get_sysinfo()
       ;;
   esac
 
+  mkdir -p ${info_system}
```

Comment:
${info_system} is ultimately derived from $(dirname $0) on lines 29-31. This value could contain spaces or other characters that need to be quoted. We use `${info_system}` without quotes in quite a few places here. That should probably be fixed.

Reply:
The inconsistency in quoting for paths is concerning. I've created https://github.com/awslabs/ecs-logs-collector/issues/36 to follow up with.